### PR TITLE
rdf-Format der Goobi-xml-Dateien in das neue mets-Format wandeln

### PR DESCRIPTION
**Pull request by arminiusdc**

### CODE REVIEW

**Comment by stweil** [comment]
Thank you for contributing this script to the community. May I suggest a different subject line (English text, see also http://chris.beams.io/posts/git-commit/)?

Convert from Goobi RDF format to METS format

Regards
Stefan

**Comment by henning-gerhardt** [comment]
It does not matter if a pull request is in German or English. I prefer a low level entry for pull request for new developers / contributors. I send him a mail about missing copyright information which it is more important than an english-speaking pull request.

**Comment by stweil** [comment]
Yes, we should help new contributors and don't scare them away. Nevertheless it would be good to have a commit history in English.

Copyright information is not strictly necessary because there are legal rules which apply automatically. If someone writes non-trivial code, he or she has the copyright.

More important is the missing license information. It is missing for other files in goobi-contrib, too (at least the first php file I was looking at did not have any). Maybe it is not too late to add a general notice that all files here are published under the GPL (I suggest GPL 3) if there is no separate license statement for a contribution. It is also possible to add something like "All contributions after 2015-09-21 are licensed under the terms of the GNU GPL, version 3 or (at your option) any later version." to the README.md file.

Stefan

File: sbb/agora2mets.pl [hunk 1]
```diff
@@ -0,0 +1,488 @@
+#!/opt/ActivePerl-5.16/bin/perl
```
Comment (stweil) [inline sbb/agora2mets.pl:1]: ActivePerl is typical for Windows environment, but typical Linux distributions use /usr/bin/perl.

Comment (arminiusdc) [inline sbb/agora2mets.pl:1]: I use ActivePerl in both environments. So I don´t interfere with system perl.

## I need install module like xml::LibXml and I can use the ppm from activeperl.

---

Armin Möller,                     Administrator
Abt. IDM Ref. 2                 Tel +49 30 266 432231
Staatsbibliothek zu Berlin PK
Potsdamer Str. 33
10772 Berlin                         armin.moeller@sbb.spk-berlin.demailto:armin.moeller@sbb.spk-berlin.de

Von: Stefan Weil [mailto:notifications@github.com]
Gesendet: Montag, 21. September 2015 17:56
An: goobi/goobi-contrib
Cc: Möller, Armin
Betreff: Re: [goobi-contrib] rdf-Format der Goobi-xml-Dateien in das neue mets-Format wandeln (#1)

In sbb/agora2mets.plhttps://github.com/goobi/goobi-contrib/pull/1#discussion_r39988000:

> @@ -0,0 +1,488 @@
> 
> +#!/opt/ActivePerl-5.16/bin/perl

ActivePerl is typical for Windows environment, but typical Linux distributions use /usr/bin/perl.

Comment (henning-gerhardt) [inline sbb/agora2mets.pl:1]: Goobi contrib tools must not be run on every environment without modification. It's fine for me.

**Comment by henning-gerhardt** [comment]
I did wrote him not only about copyright information, I included licence information too. It should be a open licence like any of CC licences or GPL. I did not prefer for contrib tools any specific free licence.

**Review by henning-gerhardt — APPROVED** [review]

**Pull request merged**